week-3/Homework/extra/cardvalidator.js

```diff
@@ -0,0 +1,28 @@
+
+
+function validateNumber(arr) {
+let validation = /(?=.{16})(?=.*[0-9]{2})(?=.([02468]$))/ 
```

Comment:
Similar to the password validator exercise, do you feel like you could explain this code?